packages/next-on-pages/templates/_worker.js/utils/http.ts

```diff
@@ -89,7 +89,7 @@ export function createRouteRequest(req: Request, path: string) {
 	applySearchParams(newUrl.searchParams, new URL(req.url).searchParams);
 
 	newUrl.pathname = newUrl.pathname
-		.replace(/^\/index.html$/, '/')
+		.replace(/\/index.html$/, '/')
```

Comment:
as before, I wonder if not forcing this to apply at the root is the correct call 🤔

Reply:
i would direct your complaints to the Pages team :)

https://developers.cloudflare.com/pages/platform/serving-pages/#route-matching

Reply:
ah! you mean this?

![Screenshot at 2023-10-29 00-14-38](https://github.com/cloudflare/next-on-pages/assets/61631103/03e87a02-f79a-41c0-a08c-8d986de36181)


yeah good point :thinking:

Reply:
yup...

Reply:
ok then yeah I think it's fine to just leave it like this 🙂